README.md

```diff
@@ -96,41 +74,41 @@ The added property `AndroidManifest.xml`. Note: You should only include the `clo
 
 
 
-### Embedding and transforming images
+### Transform and Optimize Assets
 
 Any image uploaded to Cloudinary can be transformed and embedded using powerful view helper methods:
 
 The following example generates the url for accessing an uploaded `sample` image while transforming it to fill a 100x150 rectangle:
 
     MediaManager.get().url().transformation(new Transformation().width(100).height(150).crop("fill")).generate("sample.jpg")
 
-Another example, embedding a smaller version of an uploaded image while generating a 90x90 face detection based thumbnail: 
+Another example, embedding a smaller version of an uploaded image while generating a 90x90 face detection based thumbnail:
 
     MediaManager.get().url().transformation(new Transformation().width(90).height(90).crop("thumb").gravity("face")).generate("woman.jpg")
 
-You can provide either a Facebook name or a numeric ID of a Facebook profile or a fan page.  
-             
+You can provide either a Facebook name or a numeric ID of a Facebook profile or a fan page.
+
 Embedding a Facebook profile to match your graphic design is very simple:
 
     MediaManager.get().url().type("facebook").transformation(new Transformation().width(130).height(130).crop("fill").gravity("north_west")).generate("billclinton.jpg")
-                           
+
 Same goes for Twitter:
 
     MediaManager.get().url().type("twitter_name").generate("billclinton.jpg")
 
 ### Uploading
 
-The entry point for upload operations is the `MediaManager.get().upload()` call. All upload operations are dispatched to a background queue, with 
+The entry point for upload operations is the `MediaManager.get().upload()` call. All upload operations are dispatched to a background queue, with
 a set of fully customizable rules and limits letting you choose when each upload request should actually run. Requests are automatically rescheduled to be
 retried later if a recoverable error is encountered (e.g. network disconnections, timeouts).
 
 The upload results are dispatched asynchronously using `UploadCallback`. Global callbacks can be defined, as well as specific callbacks per request.
```

Comment:
Way too much information.
Again, IMO, steps... 
After you get the user to make it work, you can explain the details.